scss/mixins/_position.scss

```
@@ -0,0 +1,15 @@
+@mixin position(
+  $pos: absolute,
+  $top: auto,
+  $right: auto,
+  $bottom: auto,
+  $left: auto,
+  $zIndex: 1
+) {
+  position: absolute;
+  top: auto;
```

Truyền biến vào thuộc tính tương ứng
```suggestion
  top: $top;
  ...
```